interview-questions.md

```diff
@@ -4,49 +4,53 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. Why would you use the method super()?
 
-  Your answer:
+  Your answer: Super is used in a constructor, required by React
```

Comment:
Almost. Super is used on Classes in general. The question is not relating to React necessarily.  And it's also not required to be used. Additionally, the question is asking WHY you would use one, and unfortunately this response does not answer it.